tests/test_jwtAuth.py

```diff
@@ -0,0 +1,90 @@
+import unittest
+import requests
+import json
+
+from pyTigerGraphUnitTest import make_connection
+from pyTigerGraph import TigerGraphConnection
+
+
+class TestJWTTokenAuth(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.conn = make_connection(graphname="Cora")
+
+
+    def test_jwtauth(self):
+        dbversion = self.conn.getVer()
+
+        if "3.9" in str(dbversion):
+            self._test_jwtauth_3_9()
+        elif "4.1" in str(dbversion):
+            self._test_jwtauth_4_1_success()
+            self._test_jwtauth_4_1_fail()
+        else:
+            pass
+        
+
+    def _requestJWTToken(self):
+        # Define the URL
+        url = f"{self.conn.host}:{self.conn.gsPort}/gsqlserver/requestjwttoken"
```

Comment:
Should we make the jwt request endpoint as a utility function?

Reply:
Good point. Maybe we should add it to the GSQL integration plan?

Reply:
That is good idea. I will create another PR for this.